src/twitch.js

```diff
@@ -79,11 +78,16 @@ TwitchCtrl.prototype.getFeaturedStreams = function(options) {
 * @param {String} options.language : only shows streams of a certain language. Permitted values are locale ID strings, e.g. {en}, {fi}, {es-mx}
 * @returns {Promise.<string, Error>} : resolves JSON data or rejects an error
 */
-TwitchCtrl.prototype.getTopStreams = function(options) {
+TwitchCtrl.prototype.getTopStreams = function(options = null) {
```

Comment:
I don't think the `= null` is required. if you were to just call `getTopStreams()` without a param, `options` inside the function would be `undefined`, which is a [falsy value](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)